tally_ho/apps/tally/views/data/center_list_view.py

```diff
@@ -60,7 +75,24 @@ class CenterListView(LoginRequiredMixin,
 
     def get(self, *args, **kwargs):
         tally_id = kwargs.get('tally_id')
+        region_id = kwargs.get('region_id', None)
+        constituency_id = kwargs.get('constituency_id', None)
+        sub_constituency_id = kwargs.get('sub_constituency_id', None)
         format_ = kwargs.get('format')
+        region_name = None
```

Comment:
optional, and I'm happy w/you not making this change

but I'd probably write this as

```python
region_name = region_id and Region.objects.get(id=region_id).name
...
```

and remove the conditionals below, it should be equivalent but less verbose

Reply:
Will do

Reply:
Done